witness/README.md

```diff
@@ -0,0 +1,44 @@
+Witnessing
+============
+
+Witnessing is an important part of any transparent ecosystem, as it ensures that
+misbehavior on the part of logs can be prevented or detected. This is done by
+ensuring that the _checkpoints_ published by logs are _consistent_, which means 
+the log is presenting a globally consistent view of its contents to everyone and
+thus isn't carrying out a _split-view attack_.
+
+In terms of checkpoints, we consider witness implementations that use the
+[format defined in this
+repository](https://github.com/google/trillian-examples/tree/master/formats/log).
+
+In terms of checking consistency, witnesses can do so in the following two 
+(non-exhaustive) ways:
+- Getting a "classical" consistency proof, assuming all they have is a log root.
```

Comment:
I think "classical" here relies on too much context that newcomers might not have. Perhaps a link to something like rfc6962 would be useful?